app/component/user/edit-biz/edit-biz.html

```diff
@@ -45,6 +45,8 @@
       ng-model="editBizCtrl.biz.url">
     </fieldset>
 
+    <menu-list biz="editBizCtrl.biz"></menu-list>
```

Comment:
This is great. Because you have the binding for biz in the menu-list controller, you can reference it as `this.biz` inside the controller.